packages/navigation/src/types.ts

```diff
@@ -0,0 +1,21 @@
+import type { NativeStackNavigationOptions } from '@react-navigation/native-stack';
+
+export interface WebScreenRule {
+  urlPattern: string;
+  title?: string;
+  presentation?: NativeStackNavigationOptions['presentation'];
+  component?: React.ElementType;
+  options?: NativeStackNavigationOptions;
```

Comment:
Is `title` and `presentation` needed here as separate keys? What about just providing them part of the `options`?

Reply:
I left it as it was. But after refactor it indeed seems better solutions (to move it to options)